webapp/channels/src/components/edit_post/edit_post.tsx

```diff
@@ -233,12 +235,20 @@ const EditPost = ({editingPost, actions, canEditPost, config, channelId, draft,
             return;
         }
 
-        const updatedPost = {
+        let updatedPost = {
             message: editText,
             id: editingPost.postId,
             channel_id: editingPost.post.channel_id,
         };
 
+        const hookResult = await actions.runMessageWillBeUpdatedHooks(updatedPost, editingPost.post);
+        if (hookResult.error && !hookResult.error.message) {
```

Comment:
I assume this is a typo, we want `hookResult.error.message` to be defined/not empty to show the error.